registration/include/pcl/registration/impl/gicp.hpp

```diff
@@ -62,10 +62,8 @@ GeneralizedIterativeClosestPoint<PointSource, PointTarget>::computeCovariances(
   }
 
   Eigen::Vector3d mean;
-  pcl::Indices nn_indecies;
-  nn_indecies.reserve(k_correspondences_);
-  std::vector<float> nn_dist_sq;
-  nn_dist_sq.reserve(k_correspondences_);
+  pcl::Indices nn_indices(k_correspondences_);
```

Comment:
Please revert changes from L65-L68 and keep changes that are related to the title